notebooks/Custom SSM - Daily Seasonality.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<p>Busy plot, lower alpha again?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-experimental/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I was actually wanting to delete this notebook, it was an example I did for discourse at one point. There's a better example of how to make your own statespace in the PyMCStateSpace docstring. I'd replace this whole notebook with one based on that.

Reply:
Sure

Reply:
I re-did this notebook, it's now called "Making a Custom Statespace Model". Have a look and see what you think.